x-pack/plugins/session_view/public/components/session_view_display_options/index.tsx

```diff
@@ -54,6 +56,21 @@ export const SessionViewDisplayOptions = ({
         ),
         key: VERBOSE_MODE_OPTION_KEY,
         checked: displayOptions?.verboseMode ? 'on' : undefined,
+        append: (
+          <EuiIconTip
+            content={
+              <FormattedMessage
+                id="xpack.sessionView.sessionViewToggle.sessionViewToggleOptionsVerboseModeTooltips"
+                defaultMessage="When verbose mode is enabled, session view displays all processes created in that session in the selected time period. When disabled, only process group leaders are rendered under the session leader process. This may hide many processes, such as shell startup, shell completion, and forks caused by builtin commands.  For more information on process groups see {VERBOSE_MODE_TOOLTIPS_LINK}"
+                values={{
+                  VERBOSE_MODE_TOOLTIPS_LINK: (
+                    <EuiLink href="https://en.wikipedia.org/wiki/Process_group">here</EuiLink>
```

Comment:
external prop should be set. see: https://elastic.github.io/eui/#/navigation/link#external-links